web_programming/get_top_billionaires.py

```diff
@@ -65,7 +68,7 @@ def get_forbes_real_time_billionaires() -> list[dict[str, int | str]]:
             "Country": person["countryOfCitizenship"],
             "Gender": person["gender"],
             "Worth ($)": f"{person['finalWorth'] / 1000:.1f} Billion",
-            "Age": years_old(person["birthDate"]),
+            "Age": str(years_old(person["birthDate"] / 1000)),
```

Comment:
If this `/ 1000` is required then please move it into the `years_old()` function and ensure that the tests pass.